src/Pim/Bundle/DataGridBundle/Extension/MassAction/MassActionDispatcher.php

```diff
@@ -39,22 +39,28 @@ class MassActionDispatcher
     /** @var MassActionParametersParser */
     protected $parametersParser;
 
+    /** @var array */
+    protected $gridsWithoutMassActionRepository;
```

Comment:
@damien-carcel because that's for the grid that don't need the mass action repository